core/domain/rights_manager.py

```diff
@@ -324,9 +324,12 @@ class Actor(object):
     def __init__(self, user_id):
         # Note that this may be None.
         self.user_id = user_id
+        self._is_admin = None
```

Comment:
Perhaps add a comment saying: "The value of None is a placeholder. This property is initialized when the first call to is_admin() is made."

Do we need to do something similar for is_moderator()?